core/block/import/common/objectcreator/task.go

```diff
@@ -32,13 +34,18 @@ func NewDataObject(ctx context.Context,
 	origin objectorigin.ObjectOrigin,
 	spaceID string,
 ) *DataObject {
+	newIdsSet := make(map[string]struct{}, len(oldIDtoNew))
```

Comment:
I've created this set to help me quickly find out which objects are created during import. It's required to handle files properly, because we don't actually import file objects, we recreate them via Upload. Maybe it's better to create file object payload in Upload function, but it requires a lot of work